ecommerce/bff/executive_education_2u/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import include, url
+from rest_framework.routers import DefaultRouter
+
+from ecommerce.extensions.basket.views import ExecutiveEducation2UAPIViewSet
+
+router = DefaultRouter()
+router.register(r'', ExecutiveEducation2UAPIViewSet, basename='executive_education_2u')
+urlpatterns = [
+    url(r'^v0/', include((router.urls, 'v0'))),
```

Comment:
[curious] why v0 vs v1?

Reply:
to stay in convention with the other bff endpoint, not sure why it started at v0.